banking/ebics/types.py

```diff
@@ -0,0 +1,66 @@
+from datetime import date
+from decimal import Decimal
+from typing import TypedDict
+
+
+class BalanceInfo(TypedDict):
+	amount: Decimal
+	currency: str
+	date: date
+
+
+class SumInfo(TypedDict):
+	amount: Decimal
+	currency: str
+	count: int
+
+
+class CurrencyAmount(TypedDict):
+	amount: Decimal
+	currency: str
+
+
+class MT940SepaInfo(TypedDict):
+	EREF: str
+	MREF: str
+	CRED: str
+	SVWZ: str
```

Comment:
```suggestion
	EREF: str | None
	MREF: str | None
	CRED: str | None
	SVWZ: str | None
```

According to my test data, these are all optional.